packages/fxa-shared/db/models/auth/index.ts

```diff
@@ -30,7 +30,7 @@ export type PayPalBillingAgreementStatusType =
   | 'Pending'
   | 'Active'
   | 'Suspended'
-  | 'Cancelled'
```

Comment:
What about all the existing entries in the database that were marked 'cancelled'?

Reply:
Adding 'Cancelled' back in for the existing entries.

Reply:
Oh, to be clear, I don't think we should have 2 of these in the database, just one. And since we may already have them with 'Cancelled', we should just retain that spelling in the database.

Reply:
Aah ok, I thought we were correcting the status from 'Cancelled' to 'Canceled' when it gets passed into `updatePayPalBA`, which would need 'Canceled' to be added in type `PayPalBillingAgreementStatusType`.